tests/Validation/ValidationRequiredIfTest.php

```diff
@@ -29,4 +29,30 @@ public function testItClousureReturnsFormatsAStringVersionOfTheRule()
 
         $this->assertSame('', (string) $rule);
     }
+
+    public function testItOnlyCallableAndBooleanAreAcceptableArgumentsOfTheRule()
```

Comment:
This test would lead to falls positives.

`expectException` will success if **one** of the 3 rules hit
it would be better to test with a data provider or separate tests.

Reply:
oh my bad, will fix it.